### DISCUSSION TITLE
Setup

### DISCUSSION THREAD

**Discussion by michaelthompson1991** [BottlecapDave/HomeAssistant-OctopusEnergy#525] [Q&A]
So I’ve attempted setting up the octopus integration which I’ve achieved but I don’t have a home mini, can this work without this? I’ve had it running an day and it’s still returning 0 for everything 🤔

**Comment by anothermort** [comment]
It will work if you have a grid meter you can access via HA (which is what I used to do before having a mini) or if not the data will be a day out.
As it will get it via the API when its available the next day. If I remeber correctly.

**Reply by BottlecapDave** [reply]
Correct, the integration should work fine without a Home Mini, but the consumption data will be a day (or sometimes more) out. Am I assuming it's just the consumption data/cost that is zero and not "everything"? The best way is to check how far back you can access the data on the website/app. You can then adjust the integration to always look that many days in the past. I have it on my list to update the sensor to look for the last "full days" worth of data so that it moves with OE's speed.

**Reply by michaelthompson1991** [reply]
Thanks! I have about 4 different values in kWh, I think 1 is peak and 1 is off peak. Then there’s also different values for the price too. I’m assuming I would group them together but there’s also not a helper in the ui. As you can tell I’m completely new to energy monitoring in HA

**Reply by BottlecapDave** [reply]
So is all of the expected data coming back now or are you still missing data? If you're wanting to use the energy dashboard, then you can follow the guide outlined in https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md#previous-day-consumption

**Reply by michaelthompson1991** [reply]
I’m not sure tbh. I’m not sure I’ve even chose the correct entities. I noticed I had a value on the energy dashboard for energy but not gas, and after changing the gas entity I now have a value for gas but not energy. Not sure if I just need to wait for it to populate or not 🤷🏼 but thanks for the link, I’ll take a look
